docs/docs/developers/development/dialogs/multifield/index.md

```diff
@@ -4,11 +4,11 @@
 
 ## Description
 
-Multifield component allows to add/reorder/remove multiple instances of a field.
+The Multifield component allows users to add, reorder or remove multiple instances of a field.
 
-In the simplest case, this is a simple form input field (e.g. TextField, TextArea) but it can also be a complex component acting as an aggregate of multiple subcomponents (e.g. address entry).
+In a simple case, this creates a basic form input field (e.g., TextField, TextArea). However, it can also create a complex component that acts as an aggregate of multiple subcomponents (such as address entries).
```

Comment:
"creates a basic form" - @pawel-boguski-ds is this correct?

Reply:
I updated this section https://github.com/websight-io/docs/pull/132/commits/265aa631ffba58ec68b754fd081a63913e5dc87f